libr/core/cmd_search.c

```diff
@@ -504,8 +493,8 @@ static int __cb_hit(RSearchKeyword *kw, void *user, ut64 addr) {
 				r_cons_printf (",");
 			}
 			char *es = escaped ? s : r_str_escape (s);
-			r_cons_printf ("{\"offset\": %"PFMT64d ",\"type\":\"%s\",\"data\":\"%s\"}",
-				base_addr + addr, type, es);
+			r_cons_printf ("{\"offset\":%"PFMT64d ",\"id\":%d,\"type\":\"%s\",\"data\":\"%s\"}",
```

Comment:
why are you reintroducing this?

Reply:
id is useless for the user, a flagname field would be more handy in any case. i just removed it because with search.in the whole id concept gets a bit confusing, and also reduces size of the output